binary_gentoo/internal/cli/build.py

```diff
@@ -125,6 +126,22 @@ def parse_command_line(argv):
                         action='store_true',
                         help='enforce installation (default: build but do not install)')
 
+    parser.add_argument('--use',
```

Comment:
Can we move this argument right before the existing `--makeopts` argument? (`--update` below already seems to be at a good spot.)